.github/workflows/main.yml

```diff
@@ -156,3 +156,5 @@ jobs:
             xgboost \
             cpp \
             include src python-package
+
+        sh ./tests/ci_build/lint_cmake.sh || true
```

Comment:
`cmakelint` still reports 157 errors on latest `master`. I'm proposing keeping this `|| true` to avoid failing builds until all of those errors have been addressed (which I'm planning to do in followup PRs).

The last PR I make for #9630 would then remove this `|| true`, so future violations of `cmakelint`'s preferences cause CI failures.